hoon/guides/tudumvc.md

```diff
@@ -0,0 +1,28 @@
++++
```

Comment:
this will not build with zola as hoon/guides/tudumvc.md  and hoon/guides/tudumvc/_index.md are conflicting paths.

Reply:
@baudtack Thank you for the guidance here - I had not considered building the product to test locally and was trying just to emulate style. Much appreciate the correction - working on it now.

Reply:
@rabsef-bicrym yeah It bites me in the butt all the time when i try do to that also. If you need a hand getting it working let me know.

Reply:
@baudtack Is there a nesting depth limit on the sidebar? I'm having trouble getting a 4th one to show - which may not be a problem - I can reconfigure things to so accommodate.

Reply:
I think there is but i'm not really sure about that. @matildepark do you know?